packages/jaspr/lib/src/framework/framework.dart

```diff
@@ -772,6 +796,11 @@ abstract class Element implements BuildContext {
           dependency.didRebuildDependent(this);
         }
       }
+      if (_observerElements != null) {
```

Comment:
```suggestion
      if (_observerElements != null && _observerElements!.isNotEmpty) {
```

Reply:
Done in https://github.com/schultek/jaspr/pull/27/commits/263281b07efbb25ac46edb8eab09c8f3c24a90f8